qiime2/sdk/typemap.py

```diff
@@ -0,0 +1,48 @@
+import random
+from itertools import combinations
+import networkx as nx
+import matplotlib.pyplot as plt
+
+import qiime 2
```

Comment:
Hmm, looks like a typo, and also appears to be unused. If that is the case, please remove.